src/NovelRT/Experimental/Graphics/Vulkan/VulkanGraphicsMemoryAllocator.cpp

```diff
@@ -0,0 +1,254 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
+// for more information.
+
+#include <NovelRT/Experimental/Graphics/Vulkan/Graphics.Vulkan.h>
+
+namespace NovelRT::Experimental::Graphics::Vulkan
+{
+    // TODO: I have no idea if this works. Lol.
+#ifdef _MSC_VER
+#pragma warning(push)
+#pragma warning(disable : 4996)
+#elif defined(__clang__)
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
+#endif
```

Comment:
_Whyyyyyyyyyyyyyyyyyy_? ;_;

Reply:
`NovelRT::Maths::Utilities::PopCount` is new but is also immediately deprecated as I'm deleting it once we're in C++20.